### ISSUE TITLE
SCM URL in pom files are wrong

### ISSUE THREAD

**Issue by menski** [zeebe-io/zeebe#2165]
```
zb-bpmn-model-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-bpmn-model                                                 
zb-db-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-db                                                                 
zb-dispatcher-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-dispatcher                                                 
zb-elasticsearch-exporter-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-elasticsearch-exporter                         
zb-exporter-api-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-exporter-api                                             
zb-gossip-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-gossip                                                         
zb-logstreams-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-logstreams                                                 
zb-msgpack-core-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-msgpack-core                                             
zb-msgpack-json-el-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-msgpack-json-el                                       
zb-msgpack-json-path-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-msgpack-json-path                                   
zb-msgpack-value-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-msgpack-value                                           
zb-protocol-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-protocol                                                     
zb-protocol-impl-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-protocol-impl                                           
zb-raft-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-raft                                                             
zb-service-container-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-service-container                                   
zb-transport-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-transport                                                   
zb-util-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zb-util                                                             
zeebe-broker-core-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zeebe-broker-core                                         
zeebe-gateway-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zeebe-gateway                                                 
zeebe-gateway-protocol-0.17.0-SNAPSHOT.pom: https://github.com/zeebe-io/zeebe/zb-parent/zeebe-gateway-protocol    
```

**Comment by Zelldon** [comment]
I see the following on the CI

```
[WARNING] 
[2019-08-27T13:06:54.134Z] [WARNING] Some problems were encountered while building the effective model for io.zeebe:zeebe-parent:pom:0.21.0-SNAPSHOT
[2019-08-27T13:06:54.134Z] [WARNING] The expression ${scm.url} is deprecated. Please use ${project.scm.url} instead.
[2019-08-27T13:06:54.134Z] [WARNING] 
```

**Comment by npepinpe** [comment]
I think this was fixed by #4998
